### PR TITLE
update Metabase to use a postgres backend

### PR DESCRIPTION
**Pull request by mtCarto**
also removes old build pod that has since been replaced by actions

### CODE REVIEW

**Review by KitArmstrong — APPROVED** [review]

File: openshift/templates/metabase/metabase.dc.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "epic-metabase"
+    "name": "mds-metabase-dc"
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase.dc.json]: whats the mds for?

File: openshift/templates/metabase/metabase.dc.json [hunk 2]
```diff
@@ -71,18 +72,47 @@
...
+        },
+        "name": "${NAME_DATABASE}-${SUFFIX}"
+      },
+      "spec": {
+        "ports": [
+          {
+            "name": "postgresql",
+            "port": 5432,
+            "protocol": "TCP",
+            "targetPort": 5432
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase.dc.json]: might want to make these ports into openshift secrets or something. not sure if they're considered to be sensitive information or not.

File: openshift/templates/metabase/metabase.dc.json [hunk 3]
```diff
@@ -176,12 +239,33 @@
...
+                  },
+                  {
+                    "name": "MB_DB_DBNAME",
+                    "value": "metabase"
                   },
                   {
-                    "name": "MB_DB_TYPE",
-                    "value": "h2"
+                    "name": "MB_DB_PORT",
+                    "value": "5432"
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase.dc.json]: here too

File: openshift/templates/metabase/metabase.dc.json (hunk 2 above)
Comment (MaxWardle) [reply openshift/templates/metabase/metabase.dc.json]: or maybe as an input param

File: openshift/templates/metabase/metabase.dc.json [hunk 4]
```diff
@@ -216,13 +294,88 @@
...
+                      "pg_isready -h localhost -p 5432"
+                    ]
+                  },
+                  "initialDelaySeconds": 20,
+                  "timeoutSeconds": 15
+                },
+                "livenessProbe": {
+                  "initialDelaySeconds": 30,
+                  "tcpSocket": {
+                    "port": 5432
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase.dc.json:360]: ^

File: openshift/templates/metabase/metabase.dc.json [hunk 5]
```diff
@@ -216,13 +294,88 @@
                 "terminationMessagePath": "/dev/termination-log",
                 "imagePullPolicy": "IfNotPresent",
                 "capabilities": {}
+              },
+              {
+                "name": "${NAME_DATABASE}",
+                "image": " ",
+                "ports": [
+                  {
+                    "containerPort": 5432,
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase.dc.json]: and here

File: openshift/templates/metabase/metabase.dc.json [hunk 6]
```diff
@@ -216,13 +294,88 @@
...
+                      }
+                    }
+                  }
+                ],
+                "readinessProbe": {
+                  "exec": {
+                    "command": [
+                      "/bin/bash",
+                      "-c",
+                      "pg_isready -h localhost -p 5432"
```
Comment (MaxWardle) [inline openshift/templates/metabase/metabase.dc.json]: ^

**Review by MaxWardle — COMMENTED** [review]
looks good :+1: , just want to have a conversation to confirm a couple things first.

**Review by MaxWardle — APPROVED** [review]

File: openshift/templates/metabase/metabase.dc.json (hunk 1 above)
Comment (mtCarto) [reply openshift/templates/metabase/metabase.dc.json]: whoops not sure how that changed back

File: openshift/templates/metabase/metabase.dc.json (hunk 2 above)
Comment (mtCarto) [reply openshift/templates/metabase/metabase.dc.json]: updating to be a param

**Pull request merged**